core/math/math_defs.h

```diff
@@ -41,6 +41,7 @@
 
 #define Math_SQRT12 0.7071067811865475244008443621048490
 #define Math_SQRT2 1.4142135623730950488016887242
+#define Math_SQRT3 1.7320508075688772935274463415059
```

Comment:
This is a trivial math change, it does not need attribution.

Reply:
Okay, thanks for letting me know, wanted to be thorough.